changelog.d/gh-8234.added

```diff
@@ -0,0 +1 @@
+feat(attribute ellipsis argument): Added support for ellipsis patterns in attribute argument position. (e.g. `#[get(...)]`)
```

Comment:
```suggestion
Rust: Added support for ellipsis patterns in attribute argument position. (e.g. `#[get(...)]`)
```

Usually we don't use the conventional commit style on changelog entries, though I'm now realizing that several recent entries do.